packages/amplication-client/src/Resource/git/AuthResourceWithGit.tsx

```diff
@@ -94,26 +99,82 @@ function AuthResourceWithGit({ resource, onDone }: Props) {
     setPopupFailed(true);
   };
   const errorMessage = formatError(error);
+
+  const [connectGitRepository, { error: errorUpdate }] = useMutation(
+    CONNECT_GIT_REPOSITORY
+  );
+
+  const [
+    createRepository,
+    { loading: createRepoLoading, error: createRepoError },
+  ] = useMutation(CREATE_GIT_REPOSITORY_IN_ORGANIZATION);
+
+  const handleRepoCreated = useCallback(
+    (data: CreateGitRepositoryInput) => {
+      createRepository({
+        variables: {
+          name: data.name,
+          gitOrganizationId: gitOrganization.id,
+          gitProvider: EnumGitProvider.Github,
+          public: data.public,
+          resourceId: resource.id,
+        },
+        onCompleted() {
+          setCreateNewRepoOpen(false);
+        },
+      }).catch((error) => {});
```

Comment:
Let's do something with that error, at least console log it.

Reply:
Done